client/my-sites/checkout/checkout/source-payment-box.jsx

```diff
@@ -134,6 +136,28 @@ class SourcePaymentBox extends PureComponent {
 		} );
 	}
 
+	renderBankOptions() {
+		const idealBanks = {
```

Comment:
Please add a comment where the list is from (i.e. https://stripe.com/docs/sources/ideal#optional-specifying-the-customers-bank)